docs/writing/posts/consulting.md

```diff
@@ -0,0 +1,207 @@
+
+I also try to make a good attempt at guessing exactly what their outcomes are and guessing the problems they run into to highlight my expertise. For example, one of the best things you can do is guess one of their problems and then tell them the solution right away. Usually, if the response is something like, "Oh yeah, we've been thinking about this for a couple of months now and we arrived at the same conclusion," then what you've been able to do effectively is to show that you understand the dream outcome and could have reduced the time to success.
+
+## The situational assessment
+
+Single-handedly the biggest piece of my own practice. The situational assessment is an opportunity for me to effectively highlight and showcase everything I know about the business. Demonstrate that I understand what the dream outcome is (reduce churn, improve performance, etc), and offer up methodology to get it done. One of these things can take two or three hours, but I find it extremely valuable for myself and as a tool for sales.
+
+I'll give an example if what I wrote [here](../../assessment.md). This is taken from Alan Weiss, and I've been able to implemented this with great success, heres some of his [examples](https://www.alanweiss.com/wp-content/uploads/Sample%20Proposals.pdf).
+
+### Structure of the Proposal
+
+- Situation: What is the current situation?
+- Objectives: What are the desired outcomes?
+- Values: Frame it in terms of the value equation.
+- Metrics: How do we measure success?
+- Timing
+- Terms and conditions
+
+Some callouts:
+
+- Metrics: If it's a struggle to measure success, it'll be hard to identify value and charge for it. You're likely looking at milestones and implementation, which results in trading time for money, rather than execution for results.
+
+- Situation: This is actually just hard, and you need to listen well and develop a skill in appraising the client's situation.
+
+## Some stuff I've learned so far
+
+### Time for money vs capital allocator
+
+The more high-level the client, the more I realized that they think in terms of capital allocation. They are not trading time for money; they are trading capital for time. And so, the exchange rate is different. Just like minimizing the time to success is important. If you can really capture the value of what you do, you'll likely always be charging too little. If you believe that you can actually drive a business outcome, for example, increase $100K ARR in 3 months, then you should be charging a percentage of that. It could be a deal if we see it as simply spending money to make money. Realy companies are throwing around so much capital in hopes of improving talent, their revenue, their product, their valuation.
```

Comment:
I realized that the more high-level the client, the more they think in terms of capital allocation. They are not trading time for money; they are trading capital for discretionary time. And so, the exchange rate is weighted more toward time, and the same principle applies to the importance of minimizing the time to success.